mpp/include/mpp_domains_misc.inc

```diff
@@ -571,12 +571,13 @@ end subroutine init_nonblock_type
       else
          info(11) = 0
       endif
+      info(12) = domain%ntiles
```

Comment:
I am concerned if the `info(11) = 0` on line 572 should be `info(12) = 0`.  Without knowing why info has 12 elements, I am unable to determine the reason.